app-crypt/ima-evm-utils/files/ima-evm-utils-1.4-test-remove-boot_aggregate.patch

```diff
@@ -0,0 +1,216 @@
+diff --git a/tests/Makefile.am b/tests/Makefile.am
```

Comment:
Are any of these patches upstreamed/have relevant bugs? If so, include links to any relevant references to the top of each (upstream submission or commit, gentoo bugs, ...)